lib/elixir/lib/module.ex

```diff
@@ -695,6 +695,21 @@ defmodule Module do
   defp simplify_signature(other, acc) when is_binary(other),  do: autogenerated(acc, :binary)
   defp simplify_signature(_, acc),                            do: autogenerated(acc, :arg)
 
+  defp simplify_struct(module) when is_atom(module) do
```

Comment:
Okay, I'll get that changed.  I also just realized that it can produce weird documentation if you have (for example) a module named `:"bar.baz"`.  It would produce documentation (for a function `get/1`) that looks like `def get(bar.baz)`.  Do you think that matters, or just leave the behavior as-is?